package.json

```diff
@@ -1,30 +1,39 @@
 {
```

Comment:
The root package json only needs to sync the metadata, the rest should be only and only the content from the original, so the whole config from prettier and stylelint has to go.

Reply:
ok